programs/address-lookup-table/src/processor.rs

```diff
@@ -0,0 +1,416 @@
+//! Program state processor
+
+use {
+    crate::error::AddressLookupError,
+    solana_program::{
+        account_info::{next_account_info, AccountInfo},
+        address_lookup_table::{
+            instruction::ProgramInstruction as AddressLookupInstruction,
+            state::{
+                AddressLookupTable, LookupTableMeta, LookupTableStatus, ProgramState,
+                LOOKUP_TABLE_MAX_ADDRESSES, LOOKUP_TABLE_META_SIZE,
+            },
```

Comment:
Is the plan to leave the state of the LUT program defined in `solana_program` to make it easier for other programs to access it?

Reply:
I'm not sure, honestly! On one hand, it would make more sense to contain everything in the program's crate like we do with SPL. On the other, the ALT stuff has already been in `solana-program` so we have to consider how we'd go about removing it. Perhaps it's appropriate to remove on a minor release?